src/main/java/org/folio/circulation/domain/CheckInProcessRecords.java

```diff
@@ -33,14 +34,15 @@ private CheckInProcessRecords(
     Loan loan,
     RequestQueue requestQueue,
     ServicePoint checkInServicePoint,
-    Request highestPriorityFulfillableRequest) {
+    Request highestPriorityFulfillableRequest, UUID loggedInUserId) {
```

Comment:
Please can the signature line either be formatted using multiple parameters up to line length or a parameter per line. I don’t mind which is used, as long as the formatting is consistent for the whole signature.

Reply:
done.